tests/network/vmi_infosource.go

```diff
@@ -98,17 +99,20 @@ var _ = SIGDescribe("Infosource", func() {
 					InfoSource: netvmispec.InfoSourceDomain,
 					MAC:        primaryInterfaceMac,
 					Name:       primaryNetwork,
+					QueueCount: network.DefaultInterfaceQueueCount,
 				},
 				{
 					InfoSource:    netvmispec.InfoSourceDomainAndGA,
 					InterfaceName: "eth1",
 					MAC:           secondaryInterface1Mac,
 					Name:          secondaryInterface1Name,
+					QueueCount:    network.DefaultInterfaceQueueCount,
 				},
 				{
 					InfoSource: netvmispec.InfoSourceDomain,
 					MAC:        secondaryInterface2Mac,
 					Name:       secondaryInterface2Name,
+					QueueCount: network.DefaultInterfaceQueueCount,
 				},
 				{
 					InfoSource:    netvmispec.InfoSourceGuestAgent,
```

Comment:
Can you also add it to this option, to express the "unknown" case?

Reply:
It was there but removed on the last push because felt it is redundant once we are omitempty
will return it